client/src/components/App/Content/LandingPage/InvestigationTable/StatusFilter/StatusFilter.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Card, Checkbox, IconButton, TextField, Typography } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import { Autocomplete } from '@material-ui/lab';
+import InvestigationMainStatus from 'models/InvestigationMainStatus';
+import { StatusFilter as StatusFilterType } from '../InvestigationTableInterfaces';
+import useStyles from './StatusFilterStyles';
+
+interface Props {
+    statuses: InvestigationMainStatus[];
+    filteredStatuses: StatusFilterType;
+    onStatusChange: (event: React.ChangeEvent<{}>, selectedStatuses: InvestigationMainStatus[]) => void;
```

Comment:
maybe onFilteredStatusesChange? onStatusChange can be misunderstood

Reply:
`onFilterChange` may be better

Reply:
Changed 😊